rclcpp/include/rclcpp/init_options.hpp

```diff
@@ -42,6 +42,16 @@ class InitOptions
   RCLCPP_PUBLIC
   InitOptions(const InitOptions & other);
 
+  /// Return `true` if logging should be initialized.
+  RCLCPP_PUBLIC
+  bool
+  initialize_logging() const;
```

Comment:
nitpick: this sounds oddly imperative, like it should actually initialize the logging when called. I know it's an accessor to this member variable, but I had to double take when reading it the first time. Maybe `should_initialize_logging` or `auto_initialize_logging` would be more obvious as a configuration rather than an action.

But also feel free to leave as-is if you want.

Reply:
done